01-js-intro-and-jquery/app.js

```diff
@@ -0,0 +1,27 @@
+var main = function() {
+    
+    $('.hidden').fadeIn(5000, function() {
+
+        $( '.alias' ).focus()
+    });
+
+    $('#btn').click( function( event ) {
+
+        $.get("http://bootcamp.aws.af.cm/welcome/yourname", function(text) {
+            $('.hidden').html(text.response);
+        });
+    });
+
+    $('#btn2').click( function( event ) {
```

Comment:
``event`` is not being used within your ``click`` method, but it should. Check this: http://api.jquery.com/event.preventdefault/

Also, have in mind using ``on`` (http://api.jquery.com/on/) instead ``click``. There's a lot of info regarding this on StackOverflow for example (search for "jQuery .on() vs .click()" and BOOM! :P).